moveit_core/transforms/include/moveit/transforms/transforms.h

```diff
@@ -37,11 +37,12 @@
 #ifndef MOVEIT_TRANSFORMS_
 #define MOVEIT_TRANSFORMS_
 
-#include <geometry_msgs/TransformStamped.h>
-#include <geometry_msgs/Pose.h>
+#include <geometry_msgs/msg/transform_stamped.hpp>
```

Comment:
It may make sense to make these changes repo wide with a find-replace all. Otherwise we will be manually making 462 changes across 91 files for just `geometry_msgs::` and 23 times across 19 files for `#include <geometry_msgs/msg/`

Reply:
strong yes!